src/index.html

```diff
@@ -5,8 +5,463 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <label class="header__logo">
+        <a href="#home" class="header__logo-pic"><img src="images/logo.png" alt="Moyo_logo"></a>
+      </label>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link is-active">
```

Comment:
keep it simple in this task
```suggestion
            <a href="#" class="nav__link is-active">
```